.editorconfig

```diff
@@ -1,5 +1,169 @@
+# https://docs.microsoft.com/en-us/visualstudio/ide/editorconfig-code-style-settings-reference
+# https://editorconfig.org
+
+# top-most EditorConfig file
 root = true
 
-[*.{cs,xaml}]
+# Default settings:
+# A newline ending every file
+# Use 4 spaces as indentation
+[*]
+insert_final_newline = false
+indent_style = space
+indent_size = 4
+
+[project.json]
+indent_size = 2
+
+# C# files
+[*.cs]
 indent_style = tab
-csharp_space_after_keywords_in_control_flow_statements = true
\ No newline at end of file
+
+# Modifier preferences
+dotnet_style_require_accessibility_modifiers = never:suggestion
+
+# New line preferences
+csharp_new_line_before_open_brace = all
```

Comment:
meh

Reply:
we can put whatever we want here - but what do we want?
https://docs.microsoft.com/en-us/visualstudio/ide/editorconfig-code-style-settings-reference?view=vs-2017#newline

Reply:
It should be newline in all projects except for the Xamarin.Forms.Xaml projects.

Reply:
We can override this for that project...